### PR TITLE
Guide to create commerce scratch org and stores

### PR DESCRIPTION
**Pull request by unandyala**
@W-12458119@

### CODE REVIEW

**Review by tarcang — DISMISSED** [review]

File: docs/setup-guide.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+6. Authorize your org with CLI
+
+    ```bash
+    sfdx force:auth:web:login -r <<INSTANCE_URL>> -a <<ORG_ALIAS>>
+
+    #####example#####
+    # sfdx force:auth:web:login -r https://login.test1.pc-rnd.salesforce.com -a mydevhub
+
+    ###### If you created a connected app ######
```
Comment (joeluong-sfcc) [inline docs/setup-guide.md:54]: Do we want to remove this example step?

**Review by joeluong-sfcc — DISMISSED** [review]

File: docs/setup-guide.md (hunk 1 above)
Comment (unandyala) [reply docs/setup-guide.md:54]: Just left it there in case anyone uses connected app

File: docs/setup-guide.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
+# Guide to create scratch Org and stores
+
+## Pre-requisites
+
+1. A Salesforce Org where you want to create scratch Orgs and stores
```
Comment (jshackell-sfdc) [inline docs/setup-guide.md:5]:
```suggestion
1. A Salesforce org where you want to create stores and that serves as a Dev Hub for creating scratch orgs. 
```

File: docs/setup-guide.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
+# Guide to create scratch Org and stores
+
+## Pre-requisites
```
Comment (jshackell-sfdc) [inline docs/setup-guide.md:3]:
```suggestion
## Prerequisites
```

File: docs/setup-guide.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
+# Guide to create scratch Org and stores
```
Comment (jshackell-sfdc) [inline docs/setup-guide.md:1]:
```suggestion
# How to Create Scratch Orgs and Stores
```

File: docs/setup-guide.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+
+## Pre-requisites
+
+1. A Salesforce Org where you want to create scratch Orgs and stores
+2. Enable Dev Hub on the Org
```
Comment (jshackell-sfdc) [inline docs/setup-guide.md:6]:
```suggestion
2. Enable Dev Hub on the org:
```

File: docs/setup-guide.md [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+    1. Navigate to Setup (Gear icon located in upper right corner)
```
Comment (jshackell-sfdc) [inline docs/setup-guide.md:7]:
```suggestion
    1. Navigate to Setup by clicking the gear icon located in upper right corner. 
```

File: docs/setup-guide.md [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+    2. Search for or navigate to Dev Hub
```
Comment (jshackell-sfdc) [inline docs/setup-guide.md:8]:
```suggestion
    2. Enter `Dev Hub` in the Quick Find box and select **Dev Hub**. 
```

File: docs/setup-guide.md [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+    3. Switch Enable Dev Hub to ‘Enabled’
```
Comment (jshackell-sfdc) [inline docs/setup-guide.md:9]:
```suggestion
    3. Click **Enable Dev Hub** so it says `Enabled`.
```

File: docs/setup-guide.md [hunk 9]
```diff
@@ -0,0 +1,107 @@
...
+
+## Setup
+
+1. Install SFDX CLI, see [Salesforce CLI Setup Guide](https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/) to learn more about the Salesforce CLI
```
Comment (jshackell-sfdc) [inline docs/setup-guide.md:13]:
```suggestion
1. Install Salesforce CLI.  You can install the CLI with either npm or with a downloadable installer for your specific operating system. See [Salesforce CLI Setup Guide](https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_setup_install_cli.htm) for more information.
```

File: docs/setup-guide.md [hunk 10]
```diff
@@ -0,0 +1,107 @@
...
+    ```bash
+    npm install --global sfdx-cli
+    ```
+2. Install commerce plugins
```
Comment (jshackell-sfdc) [inline docs/setup-guide.md:17]:
```suggestion
2. Install the Commerce plugins. 
```

File: docs/setup-guide.md [hunk 11]
```diff
@@ -0,0 +1,107 @@
...
+    ```bash
+    sfdx plugins:install @salesforce/commerce
+    ```
+3. Install shane plugins
```
Comment (jshackell-sfdc) [inline docs/setup-guide.md:21]:
```suggestion
3. Install the shane-sfdx-plugins. 
```

File: docs/setup-guide.md [hunk 12]
```diff
@@ -0,0 +1,107 @@
...
+    ```bash
+     sfdx plugins:install shane-sfdx-plugins
+    ```
+4. Verify sfdx-cli version
```
Comment (jshackell-sfdc) [inline docs/setup-guide.md:25]:
```suggestion
4. Verify that you installed Salesforce CLI correctly and see what version was installed.
```

File: docs/setup-guide.md [hunk 13]
```diff
@@ -0,0 +1,107 @@
...
+
+    ```bash
+    sfdx -v
+
+    # sample output
+    sfdx-cli/7.182.1 darwin-arm64 node-v16.17.1
+    ```
+
+5. Verify sfdx plugin versions
```
Comment (jshackell-sfdc) [inline docs/setup-guide.md:34]:
```suggestion
5. Verify that you successfully installed the required plugins and see what versions were installed.
```

File: docs/setup-guide.md [hunk 14]
```diff
@@ -0,0 +1,107 @@
...
+    sfdx plugins
+
+    # sample output
+    @salesforce/commerce 242.0.26
+    shane-sfdx-plugins 4.43.0
+    ├─ @mshanemc/sfdx-sosl 1.1.0
+    └─ @mshanemc/plugin-streaming 1.1.7
+    ```
+
+6. Authorize your org with CLI
```
Comment (jshackell-sfdc) [inline docs/setup-guide.md:46]:
```suggestion
6. Authorize your org for use with Salesforce CLI.
```

File: docs/setup-guide.md (hunk 1 above)
Comment (jshackell-sfdc) [inline docs/setup-guide.md:54]:
```suggestion
    ###### If you created your own connected app ######
```

File: docs/setup-guide.md [hunk 15]
```diff
@@ -0,0 +1,107 @@
...
+
+    #####example#####
+    # sfdx force:auth:web:login -r https://login.test1.pc-rnd.salesforce.com -a mydevhub
+
+    ###### If you created a connected app ######
+    # sfdx force:auth:web:login -r <<INSTANCE_URL>> -i <<CONSUMER_KEY/CLIENT_ID>> -a <<ORG_ALIAS>>
+    # sfdx force:auth:web:login -r https://login.test1.pc-rnd.salesforce.com -i XXXXX -a mydevhub
+    ```
+
+7. Create a scratch org
```
Comment (jshackell-sfdc) [inline docs/setup-guide.md:59]:
```suggestion
7. Create a scratch org.
```

File: docs/setup-guide.md [hunk 16]
```diff
@@ -0,0 +1,107 @@
...
+
+    ```bash
+    sfdx commerce:scratchorg:create -u <<ORG_USERNAME>> -a <<ORG_ALIAS>> -v <<DEVHUB_USERNAME>> -w 15 --json
+
+    #####example#####
+    # sfdx commerce:scratchorg:create -u demo_org_1@1commerce.com -a demo_org_1 -v devhubtest1sdb3@mydevhub.com -w 15 --json
+    ```
+
+8. Create B2B Aura store
```
Comment (jshackell-sfdc) [inline docs/setup-guide.md:68]:
```suggestion
8. Create a B2B Aura store.
```

File: docs/setup-guide.md [hunk 17]
```diff
@@ -0,0 +1,107 @@
...
+
+    ```bash
+    sfdx commerce:store:create -t 'B2B Commerce' -n <<STORE_NAME>> -o b2b -b <<BUYER_USER_EMAIL>> -u <<ORG_USERNAME>> -v <<DEVHUB_USERNAME>> --apiversion=<<API_VERSION>>
+
+    #####example#####
+    # Note: On completion the command outputs the buyer user name and password
+    # sfdx commerce:store:create -t 'B2B Commerce' -n b2bstore01 -o b2b -b b2b_aura_buyer@1commerce.com -u demo_org_1@1commerce.com -v devhubtest1sdb3@mydevhub.com --apiversion=57.0
+    ```
+
+9. Create B2B LWR store
```
Comment (jshackell-sfdc) [inline docs/setup-guide.md:78]:
```suggestion
9. Create a B2B LWR store.
```

File: docs/setup-guide.md [hunk 18]
```diff
@@ -0,0 +1,107 @@
...
+
+    ```bash
+    sfdx commerce:store:create -n <<STORE_NAME>> -o b2b -b <<BUYER_USER_EMAIL>> -u <<ORG_USERNAME>> -v <<DEVHUB_USERNAME>> --apiversion=<<API_VERSION>>
+
+    #####example#####
+    # sfdx commerce:store:create -n b2bstore02 -o b2b -b b2b_buyer@1commerce.com -u demo_org_1@1commerce.com -v devhubtest1sdb3@mydevhub.com --apiversion=57.0
+    ```
+
+10. Create B2C LWR store
```
Comment (jshackell-sfdc) [inline docs/setup-guide.md:87]:
```suggestion
10. Create a B2C LWR store.
```

File: docs/setup-guide.md [hunk 19]
```diff
@@ -0,0 +1,107 @@
...
+    ```bash
+    sfdx commerce:store:create -n <<STORE_NAME>> -o b2c -b <<BUYER_USER_EMAIL>> -u <<ORG_USERNAME>> -v <<DEVHUB_USERNAME>> --apiversion=<<API_VERSION>>
+
+    #####example#####
+    # sfdx commerce:store:create -n b2cstore01 -o b2c -b b2c_buyer@1commerce.com -u demo_org_1@1commerce.com -v devhubtest1sdb3@mydevhub.com --apiversion=57.0
+    ```
+
+## Useful Commands
+
+-   To see all the Orgs run `sfdx force:org:list -all`
```
Comment (jshackell-sfdc) [inline docs/setup-guide.md:98]:
```suggestion
-   To see all the orgs that you previously authorized, run `sfdx force:org:list -all`.
```

File: docs/setup-guide.md [hunk 20]
```diff
@@ -0,0 +1,107 @@
...
+-   If the Org session has timed out, then logout and login back
```
Comment (jshackell-sfdc) [inline docs/setup-guide.md:99]:
```suggestion
-   If the org session times out, then logout and log back in again.
```

File: docs/setup-guide.md [hunk 21]
```diff
@@ -0,0 +1,107 @@
...
+
+    ```bash
+    sfdx force:auth:logout -u <<ORG_USERNAME>>
+
+    sfdx force:auth:web:login -r <<INSTANCE_URL>> -a <<ORG_ALIAS>>
+    ```
+
+-   For more commands, see [Salesforce CLI Command Reference](https://developer.salesforce.com/docs/atlas.en-us.214.0.sfdx_cli_reference.meta/sfdx_cli_reference/cli_reference.htm)
```
Comment (jshackell-sfdc) [inline docs/setup-guide.md:107]:
```suggestion
-   For more commands, see the [Salesforce CLI Command Reference](https://developer.salesforce.com/docs/atlas.en-us.sfdx_cli_reference.meta/sfdx_cli_reference/cli_reference.htm).
```

**Review by jshackell-sfdc — CHANGES_REQUESTED** [review]
All done, @unandyala !

**Review by joeluong-sfcc — APPROVED** [review]

**Review by p-biyani — APPROVED** [review]

**Review by tarcang — APPROVED** [review]

**Review by jshackell-sfdc — APPROVED** [review]

**Pull request merged**